### PR TITLE
aquaculture-delight 翻译提交

### PR DESCRIPTION
**Pull request by Elevenlong1101**

### CODE REVIEW

**Comment by luoxueyasha** [comment]
#4099

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/en_us.json

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/ru_ru.json

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json

File: projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "item.aquaculturedelight.baked_pollock_with_carrots": "烤鳕鱼配胡萝卜",
```
Comment (luoxueyasha) [inline projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json:2]: 【要求】采用库内已有的翻译版本（#4099 ）。
俄语审完了，没有问题

File: projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "item.aquaculturedelight.baked_pollock_with_carrots": "萝卜炖绿鳕",
+  "item.aquaculturedelight.bass_stew": "鲈鱼炖",
+  "item.aquaculturedelight.buckling": "热熏鲱鱼",
+  "item.aquaculturedelight.catfish_barbecue": "	鲇鱼烤串",
```
Comment (dovisutu) [inline projects/1.18/assets/aquaculture-delight/aquaculturedelight/lang/zh_cn.json:5]:
```suggestion
  "item.aquaculturedelight.catfish_barbecue": "鲇鱼烤串",
```
怎么有个tab……疑似typo

**Review by luoxueyasha — APPROVED** [review]

**Pull request merged**